examples/retry_command.ts

```diff
@@ -0,0 +1,34 @@
+import { Command, CommandStep, Pipeline } from '../src';
+import { YamlSerializer } from '../src/serializers/yaml';
+
+class RetryCommand extends Command {
+    constructor(public retries: number, public c: Command) {
```

Comment:
```suggestion
    constructor(private readonly retries: number, command: Command) {
```

the command passed here is just the one you wrap.